packages/db/test/upload.spec.js

```diff
@@ -278,6 +278,40 @@ describe('upload', () => {
     assert.ok(userUploads.find(upload => upload.cid === sourceCid))
   })
 
+  it('lists user uploads with CAR links in partOf', async () => {
+    const contentCid = 'bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi'
+    const sourceCid = 'QmbWqxBEKC3P8tqsKc98xmWNzrzDtRLMiMPL8wBuTGsMnR'
+    const exampleCarParkUrl = 'https://carpark-dev.web3.storage/bagbaiera6xcx7hiicm7sc523axbjf2otuu5nptt6brdzt4a5ulgn6qcfdwea/bagbaiera6xcx7hiicm7sc523axbjf2otuu5nptt6brdzt4a5ulgn6qcfdwea.car'
+    const created = new Date().toISOString()
+    const name = `rand-${Math.random().toString().slice(2)}`
+    await client.createUpload({
+      user: user._id,
+      contentCid,
+      sourceCid,
+      authKey: authKeys[0]._id,
+      type,
+      dagSize,
+      name,
+      pins: [initialPinData],
+      backupUrls: [`https://backup.cid/${created}`, exampleCarParkUrl],
+      created
+    })
+
+    // Default sort {inserted_at, Desc}
+    const { uploads } = await client.listUploads(user._id, { page: 1 })
+    assert.ok(uploads.length > 0)
+    for (const upload of uploads) {
+      // backupUrls raw is private
+      assert.ok(!('backupUrls' in upload), 'upload does not have backupUrls property')
+      assert.ok(Array.isArray(upload.partOf), 'upload.partOf is an array')
+    }
+    const namedUpload = uploads.find(u => u.name === name)
+    assert.deepEqual(namedUpload.partOf, [
+      // this corresponds to `exampleCarParkUrl`
+      'ipfs://bagbaiera6xcx7hiicm7sc523axbjf2otuu5nptt6brdzt4a5ulgn6qcfdwea'
```

Comment:
I'd drop `ipfs://`, we're only going to strip it off when consuming this API and it'll never be anything different...